src/sync/client.rs

```diff
@@ -48,19 +48,6 @@ impl Client {
         Self::new_client(conn)
     }
 
-    #[cfg(unix)]
-    /// Initialize a new [`Client`] from raw file descriptor.
-    #[deprecated(since="0.8.0", note="please use `new_from_fd` instead")]
-    pub fn new(fd: RawFd) -> Client {
-        let conn = ClientConnection::new(fd);
-
-        Self::new_client(conn).unwrap_or_else(|e| {
-            panic!(
-                "client was not successfully initialized: {}", e
-            )
-        })
-    }
-
     #[cfg(unix)]
     /// Initialize a new [`Client`] from raw file descriptor.
     pub fn new_from_fd(fd: RawFd) -> Result<Client> {
```

Comment:
How about changing the name to new?

Reply:
updated.

Reply:
hmmm, looks like github is having an incident and PR's are not syncing right now. Will check back later